connector_easypost/data/product_packaging_template.xml

```diff
@@ -0,0 +1,364 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- Copyright 2016 LasLabs Inc.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+
+<openerp>
```

Comment:
(you'll need to leave the data tag instead of adding into the header like my first statement though - I was just showing you could do that. Can't do it if you're adding an update and a no update data